emukit/quadrature/kernels/integration_measures.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+
+import numpy as np
+from typing import Tuple, List
+
+
+class IntegrationMeasure:
+    """An integration measure"""
```

Comment:
```suggestion
    """An abstract class for integration measure"""
```

Reply:
do we call it abstract class if we don't use ABC?

Reply:
absolutely, abstract class as a concept is just a part of OOO and is agnostic to language/library

Reply:
ok, done.